test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/BaseCcmRule.java

```diff
@@ -38,7 +38,7 @@ public abstract class BaseCcmRule extends CassandraResourceRule {
             new Thread(
                 () -> {
                   try {
-                    ccmBridge.remove();
+                    ccmBridge.close();
```

Comment:
Why the change from `remove` to `close`?  Possible this was changed to check directory outputs after but forgot to change back?

Reply:
Now that you've explained it, it makes sense to me, +1 for keeping it this way.